app/assets/javascripts/Components/test_run_table.jsx

```diff
@@ -254,7 +254,9 @@ class TestGroupResultTable extends React.Component {
         const marksEarned = row.original["test_results.marks_earned"];
         const marksTotal = row.original["test_results.marks_total"];
         if (marksEarned !== null && marksTotal !== null) {
-          return `${marksEarned} / ${marksTotal}`;
+          const timeout_reached = marksEarned === 0 && marksTotal === 0;
```

Comment:
Oh, I don't think you need there here (I'm fine with 0/0 for individual test cases). Just the aggregated one (that appears on the same level as the "expand" icon)